src/main/java/seedu/address/commons/core/GuiSettings.java

```diff
@@ -14,27 +15,48 @@ public class GuiSettings implements Serializable {
 
     private static final double DEFAULT_HEIGHT = 600;
     private static final double DEFAULT_WIDTH = 740;
+    private static final double DEFAULT_SPLIT_PANE_DIVIDER_POSITION = 0.75;
 
-    private final double windowWidth;
-    private final double windowHeight;
+    private double windowWidth;
+    private double windowHeight;
     private final Point windowCoordinates;
+    private final boolean isMaximized;
+    private final double splitPaneDividerPosition;
 
     /**
-     * Constructs a {@code GuiSettings} with the default height, width and position.
+     * Constructs a {@code GuiSettings} with the default parameters.
      */
     public GuiSettings() {
-        windowWidth = DEFAULT_WIDTH;
-        windowHeight = DEFAULT_HEIGHT;
+        try {
+            Rectangle2D primaryScreenBounds = Screen.getPrimary().getVisualBounds();
+            windowWidth = primaryScreenBounds.getWidth() * 0.5;
+            windowHeight = primaryScreenBounds.getHeight() * 0.9;
```

Comment:
Should these magic numbers be replaced with private static variables?

Reply:
Yes, I have changed it